### ISSUE TITLE
Support for Alpharatio 

### ISSUE THREAD

**Issue by MasalaTalk** [L4GSP1KE/Upload-Assistant#261]
Hi,
Please add support for Alpharatio tracker.
Thank you for all your efforts in creating this wonderful tool.

Cheers.

**Comment by swannie-eire** [comment]
Just beaware that this is a Scene site and that scene releases needed a scene name and not the standard L4G naming convention.

scene releases should not be marked as scene unless they are fully rar'd as per pre.

**Comment by MasalaTalk** [comment]
yes, dont add support for marking any releases as scene. they have their bot for doing scene releases.

**Comment by AN1MATEK** [comment]
> Just beaware that this is a Scene site and that scene releases needed a scene name and not the standard L4G naming convention.
> 
> scene releases should not be marked as scene unless they are fully rar'd as per pre.

I think most people are interested in non-scene releases. At least that is my case. I use L4G's script with `-m` and then go from there. It still saves me a good amount of time, but requieres manual upload to the site and adding the tracker to the torrent.

I want to point out that there may be a working script for AR in an old version of PTPUploader 

https://github.com/TnS-hun/PtpUploader/blob/master/src/PtpUploader/Source/AlphaRatio.py

If that helps, I don't really know...
